### PR TITLE
Fix issue 356 - pspm_convert_time2index

### LINKED ISSUE #356: Renaming pspm_time2index
PsPM has a series of converting functions, like
* `pspm_convert_area2diameter`
* `pspm_convert_au2unit`
* `pspm_convert_pixel2unit`

However, `pspm_time2index` has not followed this rule, which may need to be generalised.

### PR DESCRIPTION
**Pull request by teddychao**
Fixes #356

Changes proposed in this pull request:
- Update keyword `'b'` to keyward `'c'` indicating combined, which was fixing some outdated code.
- Generalise `pspm_convert_xxx` functions
- Document `pspm_convert_xxx` functions
---

### Updates for each file
- `pspm_cfg_run_pp_heart_data`
  - Update `pspm_hb2hp` to `pspm_convert_hb2hp`.
- `pspm_convert_area2diameter`, `pspm_convert_au2unit`, `pspm_convert_gaze_distance`, `pspm_convert_pixel2unit`, `pspm_convert_unit`, `pspm_convert_visangle2sps`, `pspm_convert_visangle2sps_core`
  - Convert to LF and UTF-8.
- `pspm_hb2hp`
  - Renamed to `pspm_convert_hb2hp` and convert to LF and UTF-8.
- `pspm_dcm`, `pspm_split_sessions`
  - Update to use `pspm_convert_time2index` rather than `pspm_time2index`.
- `pspm_find_valid_fixations`, `pspm_get_eyelink`, `pspm_get_viewpoint`, `pspm_load_data`, `pspm_pupil_correct_eyelink`
  - Update the keyword from `settings.lateral.full.b` to `settings.lateral.full.c`.
- `pspm_glm`
  - Update `pspm_time2index` to `pspm_convert_time2index`
- `pspm_ppu2hb`
  - Renamed to `pspm_convert_ppu2hb`.
- `pspm_butter_test`, `pspm_filtfilt_test`, `pspm_hb2hp_test`
  - Update description text.

### Updates for pspm_init
Now all warning and help text are saved in `pspm_text`. This is easier to modify the text and avoid missing space between words.

### CODE REVIEW

**Comment by dominikbach** [comment]
It seems that the "convert" functions are data conversion functions which users might like to use directly for data processing, whereas time2index is a in internal convenience function; hence I'd suggest to keep the name?

**Comment by dominikbach** [comment]
But it seems plausible to rename the other data conversion functions, such as hb2hp etc.

**Comment by dominikbach** [comment]
However, there are many more of this sort.

**Review by dominikbach — CHANGES_REQUESTED** [review]
see comments.

**Comment by teddychao** [comment]
The function `normrnd` is used by some test functions, like `pspm_load1_data`, but it requires statistics and machine learning toolbox. The init function should make this clear.

**Review by dominikbach — APPROVED** [review]

File: src/pspm_convert_area2diameter.m [hunk 1]
```diff
@@ -1,98 +1,101 @@
...
+    diam = cell(numel(chan), 1);
+    for i = 1:numel(chan)
+      [~, ~, data] = pspm_load_data(fn, chan{i});
+      diam{i} = data{1};
+      diam{i}.data = 2.*sqrt(diam{i}.data./pi);
+
+      % replace metric values
+      diam{i}.header.units = ...
+      regexprep(data{1}.header.units, ...
+      '(square)?(centi|milli|deci|c|m|d)?(m(et(er|re))?)(s?\^?2?)', ...
```
[the remark this answers is not in the document]
Comment (teddychao) [reply src/pspm_convert_area2diameter.m]: @jbrochar I agree, however I have no idea about what it means either, it is written by Tobias...

File: src/pspm_convert_pixel2unit.m [hunk 2]
```diff
@@ -1,185 +1,187 @@
...
-% $Id: pspm_find_valid_fixations.m 512 2017-12-15 13:13:08Z tmoser $
-% $Rev: 512 $
-global settings;
-if isempty(settings), pspm_init; end
-sts = -1;
-
-
-if nargin < 6
+  height, distance, options)
+
```
[the remark this answers is not in the document]
Comment (teddychao) [reply src/pspm_convert_pixel2unit.m]: Yes, there have been no blank lines in this function now.

**Pull request merged**